.clang-format

```diff
@@ -21,3 +21,5 @@ NamespaceIndentation: None
 SpaceBeforeAssignmentOperators: true
 Standard: Cpp11
 UseTab: Never
+WhitespaceSensitiveMacros:
```

Comment:
I believe this requires bumping our required ClangFormat version from clang-format-7 to clang-format-11.

```
YAML:25:3: error: unknown key 'WhitespaceSensitiveMacros'
  - bsoncxx_pragma
  ^
Error reading .clang-format: Invalid argument
```

Reply:
Removed as unnecessary.